src/views/v2/widget-list.php

```diff
@@ -0,0 +1,8 @@
+<?php
+/*
+ * Temporary Display of V2 List Widget
+ */
+?>
+<div>
+	<?php echo "Widget Name: {$name}<br>"; ?>
+</div>
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '"Widget Name: {$name}<br>"'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '"Widget Name: {$name}<br>"'.

<sub>hash: 89b84631f0c216d3af11057bea9f7ac6</sub>